front/src/components/boxs/todolist/EditTodolistBox.jsx

```diff
@@ -0,0 +1,79 @@
+import { Component } from 'preact';
+import { Text } from 'preact-i18n';
+import { connect } from 'unistore/preact';
+import BaseEditBox from '../baseEditBox';
+
+const EditTodolistBox = ({ children, ...props }) => (
+  <BaseEditBox {...props} titleKey="dashboard.boxTitle.todolist">
+    <div class="form-group">
+      <label>
+        <Text id="dashboard.boxes.todolist.editNameLabel" />
+      </label>
+      <input type="text" onChange={props.updateName} class="form-control" value={props.name} />
+    </div>
+    <div class="form-group">
+      <label>
+        <Text id="dashboard.boxes.todolist.editProjectLabel" />
+      </label>
+      <select onChange={props.updateTodolist} class="form-control">
+        <option>
+          <Text id="global.emptySelectOption" />
+        </option>
+        {props.todolist &&
+          props.todolist.map(todolist => (
+            <option selected={todolist.id === String(props.box.todolist_id)} value={todolist.id}>
```

Comment:
Do you have a reason to use String construction instead of String wrapper?
`${props.box.todolist_id}`?

Reply:
It's a mistake on my side